xchange-stream-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingRawService.java

```diff
@@ -73,7 +74,16 @@ private Object GetEventSubscriptionData(String channelName, boolean isSubscribe,
       case ORDERBOOK:
         {
           CurrencyPair currencyPair = (CurrencyPair) args[0];
-          return new CexioWebSocketOrderBookSubscriptionData(currencyPair, isSubscribe);
+          int depth = 0;
+          if (args[1] != null && args[1] instanceof Object[] && ((Object[]) args[1]).length > 0) {
+            Object[] objArgs = (Object[]) args[1];
```

Comment:
Why is this second arg an array?

Reply:
It was the result of passing a vararg parameter, inside another varargs parameter. Yes, you are right, not needed and I already pushed a change to make things easier and clearer.